src/current/cockroachcloud/connect-to-your-cluster.md

```diff
@@ -63,132 +63,77 @@ Azure Private Link is not yet available for [CockroachDB {{ site.data.products.d
-    {{site.data.alerts.end}}
-
-    You are now connected to the built-in SQL client, and can now run [CockroachDB SQL statements]({% link cockroachcloud/learn-cockroachdb-sql.md %}).
-
-  </section>
+    A welcome message displays:
 
-  <section class="filter-content" markdown="1" data-scope="connection-string">
-
-To connect to your cluster with your application, use the connection string provided in the Console:
-
-1. Select **Mac**, **Linux**, or **Windows** to adjust the commands used in the next steps accordingly.
-
-    <div class="filters clearfix">
-        <button class="filter-button page-level" data-scope="mac">Mac</button>
-        <button class="filter-button page-level" data-scope="linux">Linux</button>
-        <button class="filter-button page-level" data-scope="windows">Windows</button>
-    </div>
-
-1. In your terminal, run the first command from the dialog to create a new `certs` directory on your local machine and download the CA certificate to that directory:
-
-    {% include cockroachcloud/download-the-cert.md %}
-
-1. If you [established a private connection using VPC Peering or AWS PrivateLink](#establish-gcp-vpc-peering-or-aws-privatelink), click **VPC Peering** or **PrivateLink** to connect privately.
-
-1. Copy the connection string provided in the Console, which will be used to connect your application to CockroachDB {{ site.data.products.cloud }}:
-
-    <section class="filter-content" markdown="1" data-scope="mac">
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    'postgresql://<user>@<cluster-name>-<short-id>.<region>.<host>:26257/<database>?sslmode=verify-full&sslrootcert='$HOME'/Library/CockroachCloud/certs/<cluster-name>-ca.crt'
     ~~~
-
-    </section>
-
-    <section class="filter-content" markdown="1" data-scope="linux">
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    'postgresql://<user>@<cluster-name>-<short-id>.<region>.<host>:26257/<database>?sslmode=verify-full&sslrootcert='$HOME'/Library/CockroachCloud/certs/<cluster-name>-ca.crt'
+    #
+    # Welcome to the CockroachDB SQL shell.
+    # All statements must be terminated by a semicolon.
+    # To exit, type: \q.
+    #
     ~~~
 
-    </section>
-
-    <section class="filter-content" markdown="1" data-scope="windows">
+    You are now connected to the built-in SQL client, and can now run [CockroachDB SQL statements]({% link cockroachcloud/learn-cockroachdb-sql.md %}).
 
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    "postgresql://<user>@<cluster-name>-<short-id>.<region>.<host>:26257/<database>?sslmode=verify-full&sslrootcert=$env:appdata\CockroachCloud\certs\$<cluster-name>-ca.crt"
-    ~~~
+  </section>
 
-    </section>
+  <section class="filter-content" markdown="1" data-scope="connection-string">
```

Comment:
The tab choice for this data scope reads "General Connection String", but in the UI it's "Connection String".

Reply:
Done, great catch. I verified that other instances of "General connection string" are referring to the Serverless modal.